src/metax_api/migrations/0026_auto_20200917_1615.py

```diff
@@ -0,0 +1,88 @@
+# Generated by Django 3.0.7 on 2020-09-17 12:04
+
+from django.db import migrations
+import logging
+
+_logger = logging.getLogger(__name__)
+
+def change_emails(apps, schema_editor):
+    CatalogRecord = apps.get_model('metax_api', 'CatalogRecord')
+
+    for cr in CatalogRecord.objects.all():
+        import json
```

Comment:
I know this is merged already, for future minor improvement: `import json` is unnecessarily re-imported on every iteration, could be on the first line below function definition:
```
def change_emails(apps, schema_editor):
    import json
```

Reply:
True! I needed to get this to test-metax in order to test sending emails, but can fix it still :)